prediction_market_agent/db/db_storage.py

```diff
@@ -24,27 +28,50 @@ def _initialize_db(self) -> None:
         """
 
         # trick for making models import mandatory - models must be imported for metadata.create_all to work
-        logger.debug(f"tables being added {LongTermMemories}")
+        logger.debug(f"tables being added {LongTermMemories} {Prompt}")
         SQLModel.metadata.create_all(self.engine)
 
-    def save_multiple(
+    def save_multiple(self, items: list[Any]) -> None:
```

Comment:
Are you sure that `Any`thing is savable in the DB?

Reply:
In `test_save_prompt` you are building `Prompt` class and then saving it:

```
prompt_text = "abc"
prompt = Prompt(prompt=prompt_text, datetime_=utcnow())
db_storage_test.save_multiple([prompt]
```

But if this is typed as `Any`, it means that even `db_storage_test.save_multiple(["hello", 123, 42.0]` should work,  but that will error out, won't it?

Reply:
It's typed as `Any` since it's a generic function for saving any items (hence the `save_multiply` name).
Do you have a suggestion about how to make this better?

Reply:
I see your reasoning now.

Typing should reflect how/if the code will work. `typing.Any` is literally anything, but afaik this function will fail if you try to save anything that isn't a SQLModel's subclass.

So 

```suggestion
    def save_multiple(self, items: list[SQLModel]) -> None:
```

looks correct. It can save any SQLModel's data.

Reply:
done